src/modules/Admin/components/ListBooking/ListBooking.tsx

```diff
@@ -0,0 +1,285 @@
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import { RiDeleteBin6Line } from "react-icons/ri";
+import MoreHorizIcon from "@mui/icons-material/MoreHoriz";
+import {
+  Box,
+  Button,
+  MenuItem,
+  Modal,
+  Select,
+  Stack,
+  Typography,
+  TablePagination,
+} from "@mui/material";
+import { useEffect, useState } from "react";
+import axios from "axios";
+import { bookingUrl } from "../../../../constants/End_Points";
+import { toast } from "react-toastify";
+import styled from "@emotion/styled";
+import { tableCellClasses } from "@mui/material/TableCell"; // for border rows
+import NoData from "../../../Shared/components/NoData/NoData";
+import DeleteImg from "../../../../assets/images/delete.png";
+
+// Type for bookingData
+interface BookingData {
+  _id: string;
+  userName: string;
+  status: string;
+  totalPrice: number;
+  startDate: string;
+  endDate: string;
+  createdAt: string;
+  user: {
+    userName: string;
+  };
+}
+
+export default function Facilities() {
+  const [booking, setBooking] = useState<BookingData[]>([]);
+  const [openDelete, setOpenDelete] = useState(false); //delete modal
+  const [bookingId, setBookingId] = useState<string>("");
+  const [page, setPage] = useState(0);
+  const [rowsPerPage, setRowsPerPage] = useState(10);
+  const [totalCount, setTotalCount] = useState(0); // Total bookings count
+
+  const handleCloseDelete = () => setOpenDelete(false);
+  const handleOpenDelete = (id: string) => {
+    setBookingId(id);
+    setOpenDelete(true);
+  };
+
+  // Modal styling
+  const style = {
+    position: "absolute" as "absolute",
+    top: "50%",
+    left: "50%",
+    transform: "translate(-50%, -50%)",
+    width: 500,
+    bgcolor: "background.paper",
+    border: "none",
+    boxShadow: 24,
+    borderRadius: 2,
+    p: 4,
+  };
+
+  // Get all bookings
+  const getBooking = async () => {
+    try {
+      const response = await axios.get(
+        `${bookingUrl.getAllBooking}?page=${page}&limit=${rowsPerPage}`,
+        {
+          headers: { Authorization: localStorage.getItem("token") || "" },
```

Comment:
As I suggested before we can create an axios interceptor to add the needed headers instead of needing to write them in every api call.